src/components/ContactList/ContactList.jsx

```diff
@@ -1,22 +1,22 @@
 import PropTypes from 'prop-types';
-import s from './ContactList.module.css';
+import styles from './ContactList.module.css';
 import { useDispatch } from 'react-redux';
-import { deleteContactOperation } from '../../redux/actions';
+import { deleteContact } from '../../redux/actions';
 
 export default function ContactList({ contactsData }) {
   const dispatch = useDispatch();
 
   const deleteContactClick = id => {
-    dispatch(deleteContactOperation(id));
+    dispatch(deleteContact(id));
   };
 
   return (
-    <ul className={s.list}>
+    <ul className={styles.list}>
       {contactsData.map(({ id, name, number }) => (
-        <li className={s.item} key={id}>
-          <p className={s.name}>{name}</p>
-          <p className={s.number}>{number}</p>
-          <button className={s.button} onClick={() => deleteContactClick(id)}>
+        <li className={styles.item} key={id}>
+          <p className={styles.name}>{name}</p>
+          <p className={styles.number}>{number}</p>
+          <button className={styles.button} onClick={() => deleteContactClick(id)}>
```

Comment:
What will happen if we use `onClick={deleteContactClick}`?

Reply:
It will lead to error "DELETE https://61546f8d2473940017efae5d.mockapi.io/contacts/[object%20Object] 404 (Not Found)".

Reply:
Should be `(id) => deleteContactClick(id)`